### PR TITLE
Add note when import filecoin private key, also make desc link clickable

### PR DESCRIPTION
**Pull request by cypt4**
Resolves https://github.com/brave/brave-browser/issues/23523
Resolves https://github.com/brave/brave-browser/issues/23531
The same note as when export filecoin acc is added
Plaintext link changed to anchor

Resolves 

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:
1. Try to export filecoin account private key
2. Try to import filecoin account
3. Note with correct link should be shown

### CODE REVIEW

File: components/brave_wallet_ui/components/desktop/popup-modals/account-settings-modal/index.tsx [hunk 1]
```diff
@@ -200,7 +204,13 @@
             </WarningWrapper>
             {showPrivateKey && account.coin === BraveWallet.CoinType.FIL &&
             <WarningWrapper>
-               <WarningText>{getLocale('braveWalletFilExportPrivateKeyFormatDescription')}</WarningText>
+               <WarningText>
+                 {filPrivateKeyFormatDescriptionTextParts.beforeTag}
+                  <a target='_blank' href={FILECOIN_FORMAT_DESCRIPTION_URL}>
```
[the remark this answers is not in the document]
Comment (yrliou) [reply components/brave_wallet_ui/components/desktop/popup-modals/account-settings-modal/index.tsx:209]: Not sure why it's being marked as resolved automatically by the bot.
I know it was a requirement before but not sure what's the current policy, cc @diracdeltas.

[the remark this answers is not in the document]
Comment (yrliou) [reply components/brave_wallet_ui/components/desktop/popup-modals/account-settings-modal/index.tsx:209]: Not sure why it's being marked as resolved automatically by the bot.
I know it was a requirement before but not sure what's the current policy, cc @diracdeltas.

File: components/resources/wallet_strings.grdp [hunk 2]
```diff
@@ -212,7 +212,8 @@
   <message name="IDS_BRAVE_WALLET_ICON_URL" desc="Visible assets modal Icon URL label">Icon URL</message>
   <message name="IDS_BRAVE_WALLET_ADD_ASSET" desc="Asset Panel add asset button">Add asset</message>
   <message name="IDS_BRAVE_WALLET_ACCOUNT_SETTINGS_DISCLAIMER" desc="Account settings modal warning">WARNING: Never share your recovery phrase. Anyone with this phrase can take your assets forever.</message>
-  <message name="IDS_BRAVE_WALLET_FIL_EXPORT_PRIVATE_KEY_FORMAT_DESCRIPTION" desc="Filecoin private key format description">NOTE: Private key is provided as a hex-encoded JSON that contains private key and protocol information. See https://lotus.filecoin.io/lotus/manage/lotus-cli/#lotus-wallet-import for more info.</message>
+  <message name="IDS_BRAVE_WALLET_FIL_EXPORT_PRIVATE_KEY_FORMAT_DESCRIPTION" desc="Filecoin private key format description">NOTE: Private key is provided as a hex-encoded JSON that contains private key and protocol information. <ph name="LINK_START">$1<ex>{</ex></ph>Learn more<ph name="LINK_END">$2<ex>}</ex></ph>.</message>
```
Comment (yrliou) [inline components/resources/wallet_strings.grdp:215]: nit: Why is { and } being put as example values of $1 and $2? Do we actually need the ex for link case here?

**Review by yrliou — APPROVED** [review]
LGTM

**Comment by srirambv** [comment]
Verification passed on 

Brave | 1.42.9 Chromium: 103.0.5060.53 (Official Build) nightly (64-bit) 
-- | --
Revision | `a1711811edd74ff1cf2150f36ffa3b0dae40b17f-refs/branch-heads/5060@{#853}`
OS | Windows 11 Version 21H2 (Build 22000.739)

- Verified steps from PR
- Verified Lotus support link now shows as `Learn more` and is clickable
- Verified the hex encode message is shown when clicked on `Show key`

https://user-images.githubusercontent.com/17010094/174830405-97302a78-3a96-45f4-b1d7-a9bbe80310bf.mp4

**Pull request merged**